shell/initialize/entry-helpers.js

```diff
@@ -53,232 +47,25 @@ export const loadDebugger = (vueApp) => {
   }
 };
 
-/**
- * Handle errors with a redirect
- * @param {*} context
- * @param {*} message
- */
-const errorRedirect = (context, message) => {
-  context.$store.commit('setError', { error: new Error(message) });
-  context.$router.replace('/fail-whale');
-};
-
-/**
- * TODO: Define this logic use case
- * @param {*} fn
- * @param {*} context
- * @returns
- */
-export const promisify = (fn, context) => {
-  let promise;
-
-  if (fn.length === 2) {
-    console.warn('Callback-based fetch or middleware calls are deprecated. Please switch to promises or async/await syntax'); // eslint-disable-line no-console
-
-    // fn(context, callback)
-    promise = new Promise((resolve) => {
-      fn(context, (err, data) => {
-        if (err) {
-          context.error(err);
-        }
-        data = data || {};
-        resolve(data);
-      });
-    });
-  } else {
-    promise = fn(context);
-  }
-
-  if (promise && promise instanceof Promise && typeof promise.then === 'function') {
-    return promise;
-  }
-
-  return Promise.resolve(promise);
-};
-
 export const globalHandleError = (error) => Vue.config.errorHandler && Vue.config.errorHandler(error);
 
-/**
- * Render function used by the router guards
- * @param {*} to Route
- * @param {*} from Route
- * @param {*} next callback
- * @param {*} app
- * @returns
- */
-async function render(to, from, next) {
```

Comment:
I slowly removed more and more sections of this unitl I realized it wasn't adding value any longer. 

Ultimately the biggest thing this did was call `mount()` (passed to it inside mountApp) after it resolved. This wasn't very clear until I removed more of the implementation.

Reply:
This is a great removal - `render()` contains a lot of complexity and has traditionally been the source of a great deal of confusion in trying to understand the "why" of its existence.

Reply:
Oh yes please, I was agonizing with this one